app/src/main/java/com/rafraph/pnineyHalachaHashalem/myAudio.java

```diff
@@ -522,4 +802,31 @@ public void onRestoreInstanceState(Bundle savedInstanceState) {
         super.onRestoreInstanceState(savedInstanceState);
         serviceBound = savedInstanceState.getBoolean("ServiceState");
     }
-}
+
+    @Override
+    public void onItemSelected(AdapterView<?> adapterView, View view, int i, long l) {
+        String choice =adapterView.getItemAtPosition(i).toString();
+        float sp=1f;

```

Comment:
In this file the following section exist several times, maybe with some small changes.
Can you make it a function and use this function in all places?

Reply:
שיניתי למרות שלא נראה לי קריטי

Reply:
נכון זה לא קריטי, אבל בגלל ש-90% מהזמן קוראים קוד לעומת 10% במקרה הטוב שבו כותבים קוד. חשוב מאוד לשמור על קוד קריא ומובן וקצר